examples/Camera.ipynb

```diff
@@ -0,0 +1,506 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "31b6590e-1332-4e5a-87fd-7fc90867f2d8",
+   "metadata": {},
+   "source": [
+    "# 🎬 Camera (and Director)\n",
+    "\n",
+    "The `GraphCamera` and `GraphDirector` and [behaviors](./Behaviors.ipynb) observe and\n",
+    "control the current contents of the viewport. While the 2D and 3D APIs are very similar,\n",
+    "there are some subtle differences."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fdaed39e-214f-4f95-a975-f2f1406b1fba",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "if __name__ == \"__main__\" and \"pyodide\" in __import__(\"sys\").modules:\n",
+    "    %pip install -q -r requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d056735-7b5e-49a7-97fa-ec4419b349cb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import ipyforcegraph.behaviors as B\n",
+    "import ipyforcegraph.graphs as G\n",
+    "import ipylab as L\n",
```

Comment:
This fails in `lite` because `ipylab` is not available.  Should we add it to `requirements.txt` or add it as a separate `pip install`?

I added `ipylab ; platform_machine == "wasm32"` to `requirements.txt` and it worked.

Reply:
good catch, 98627f5